docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,52 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(fs-legacy-azure-migration)=
+## Migration to native Azure implementation
```

Comment:
Term "native Azure implementation" is wrong .. we need to reword here and below. Same for the other .. maybe just use the title of the doc

Azure Storage file system support

https://trino.io/docs/current/object-storage/file-system-azure.html

.. and expand to something along the lines of the content of that doc:

Trino includes a native implementation to access [Azure Storage](https://learn.microsoft.com/en-us/azure/storage/) with a catalog using the Delta Lake, Hive, Hudi, or Iceberg connectors.